networkx/algorithms/tree/recognition.py

```diff
@@ -157,6 +157,22 @@ def is_forest(G):
     b : bool
         A boolean that is True if `G` is a forest.
 
+    Raises
+    ------
+    NetworkXPointlessConcept
+        If `G` is empty.
+
+    Examples
+    --------
+    >>> G= nx.Graph()
+    >>> G.add_nodes_from([1,2,3,4,5])
```

Comment:
I think this line can be removed, since all of the nodes are covered in the `add_edges_from` and the node order is not important.

```suggestion
```

Reply:
The same goes for the `is_tree` example as well!

Reply:
Sure, I have made the changes.